README.md

```diff
@@ -146,7 +157,8 @@ var message = {
 };
 ```
 
-Until we make changes, you'll need to have at least one subscription before you can send on a channel.
+Currently, you'll need to have at least one subscription activated
+before you can send on a channel.
```

Comment:
Oooh! I have a pattern for this now as part of `nteract/nteract`.

They _can_ subscribe then send. However, they can also wrap sending in as part of their Observable:

```
  // On subscription, send the message
  return Rx.Observable.create(observer => {
    const subscription = original.subscribe(observer);
    channels.shell.next(executeRequest);
    return subscription;
  });
```

We do need to make our own Subject type, as suggested by @jayphelps, but I haven't been able to come back to it yet.